packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -35,6 +40,11 @@ class EmbeddedContentConsumer extends React.Component {
     // Mount the provided source as the application into the content wrapper.
     this.xfcFrame = Consumer.mount(this.embeddedContentWrapper, this.props.src, this.props.options);
 
+    // Notify that the consumer frame has mounted.
+    if (this.props.onMount) {
+      this.props.onMount(this);
```

Comment:
I'd recommend just passing back the xfcFrame

Reply:
@mhemesath can you provide your reasoning for this? I understand that the xfcFrame would be indirectly exposed via the embeddedContentConsumer Instance. However, the component already provides first class support to handle events from the provider. It felt odd to expose the xfc frame and not abstract it away.

Reply:
IMO the lifecycle events are first-class events that many consumers of XFC want to know about and the props are convenience methods to set them.  If someone needs access to custom events IMO it makes sense to just hand over XFC as its cleaner and allows that object to be passed up to a higher level so that it can be more directly integrated into the state manager (rather than be confined at the component level).